### DISCUSSION TITLE
CSS not linking with HTML

### DISCUSSION THREAD

**Discussion by ChielerLi** [community/community#31107] [New to GitHub] [answered]
Hello everyone,
I'm relatively new to coding and Github.
I've created a website with an index.html and a style.css and have tried to connect them, but have failed.

Please help.

https://github.com/ChielerLi/Chieler-s-web

**Comment by ihnatenkoo** [comment]
Hello, try it
`<link rel="stylesheet" href="./style.css">`

**Comment by ihnatenkoo** [comment, marked as answer]
Your code:
`<link rel="stylesheet" href="./style.css}">`
Must be:
`<link rel="stylesheet" href="./style.css">`

You need to delete bracket after ".css"

**Reply by ChielerLi** [reply]
Hello! Thank you so much. This took me wayy too long.

**Reply by ihnatenkoo** [reply]
You're welcome. Good luck!

**Comment by DJayJelks7** [comment]
I hope everyone is well out there.

I am having an issue linking my external CSS file to my HTML file:

[(https://github.com/DJayJelks7/Party_Tips)]

I have read through the fixes in the code on your thread and tried all of them, but the CSS file is not being recognized for some reason.

Can someone please help me out?

Thank you,

DJay

**Reply by ihnatenkoo** [reply]
Hello. Can you create a new discussion with your issue?

**Reply by DJayJelks7** [reply]
Hello, ihnatenkoo. Thank you for your response. I've restarted the repository, thinking that I may have made some missteps in the original repository. 

Here is a link to my HTML and CSS folders: https://github.com/DJayJelks7/party_tips/tree/main. 

Is it okay to have the two files in separate folders? I use VS code, but the CSS file is not recognized when downloading the HTML file with the external CSS linked.

Thank you.

DJay

**Reply by ihnatenkoo** [reply]
If you want to use css file from **css folder** the path in index html must be:
`<link rel="stylesheet" href="../css/styles.css" />`

**Reply by ihnatenkoo** [reply]
If you want to use css file from **root folder** the path in index html must be:
`<link rel="stylesheet" href="../styles.css" />`

**Reply by DJayJelks7** [reply]
I tried both code strings and neither work?

This is so strange. Can I not use github and vs code in this way? When I used internal CSS in the same file, it worked, but linking externally does not seem to. 

Example of the error code: Failed to load resource: net::ERR_FILE_NOT_FOUND
Can you try? 

Thank you.

**Reply by DJayJelks7** [reply]
> If you want to use css file from **css folder** the path in index html must be: `<link rel="stylesheet" href="../css/styles.css" />`

I have moved the index.html to the root and created a styles folder for the styles.css. I added the background color of Blue, but I am still receiving the same result of the styles.css not readable.

[(https://github.com/DJayJelks7/party_tips.github.io)]

Help!

Thank you,

DJay.
